builtin/game/chatcommands.lua

```diff
@@ -14,16 +14,13 @@ function core.register_chatcommand(cmd, def)
 end
 
 if core.setting_getbool("mod_profiling") then
-	local tracefct = profiling_print_log
+	core.register_chatcommand("dump_mod_profile", {
```

Comment:
You should at the very least document this change in the PR description. And I don't agree with the description. It should be more like "Dump mod profiling data to log".